### ISSUE TITLE
Screen reader capability throughout platform

### ISSUE THREAD

**Issue by mbakeryo** [mlibrary/heliotrope#260]
As a general public user, I should be able to use a screen reader to navigate the entire platform.

Important for furthering our accessibility mission.

**Comment by mbakeryo** [comment]
@jmcglone Do we need this ticket or do you think it should be wrapped into whatever comes out of the functional accessibility audit?

**Comment by jmcglone** [comment]
@mbakeryo this can def. be closed and any tickets from the functional accessibility audit will be more specific than this one.
